src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReplyToReferralHelper.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.et.common.model.bulk.types.DynamicFixedListType;
+import uk.gov.hmcts.et.common.model.bulk.types.DynamicValueType;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+
+import java.util.stream.Collectors;
+
+@Slf4j
+@Service
```

Comment:
If this is a service, it needs moving to the appropriate package

Reply:
And renaming

Reply:
It looks like this shouldn't be a Spring bean - it only has one static method